projects/packages/forms/src/dashboard/inbox/response.js

```diff
@@ -208,6 +234,164 @@ const InboxResponse = ( { response, loading, onModalStateChange } ) => {
 		}
 	}, [ onModalStateChange, setIsPreviewModalOpen, setIsImageLoading ] );
 
+	const handleMarkAsSpam = useCallback( async () => {
+		setIsMarkingAsSpam( true );
+		await markAsSpamAction.callback( [ response ], { registry } );
+		setIsMarkingAsSpam( false );
+		onActionComplete?.( response.id.toString() );
+	}, [ response, registry, onActionComplete ] );
+
+	const handleMarkAsNotSpam = useCallback( async () => {
+		setIsMarkingAsNotSpam( true );
+		await markAsNotSpamAction.callback( [ response ], { registry } );
+		setIsMarkingAsNotSpam( false );
+		onActionComplete?.( response.id.toString() );
+	}, [ response, registry, onActionComplete ] );
+
+	const handleMoveToTrash = useCallback( async () => {
+		setIsMovingToTrash( true );
+		await moveToTrashAction.callback( [ response ], { registry } );
+		setIsMovingToTrash( true );
```

Comment:
This needed to be `false`, [fixed in follow-up](https://github.com/Automattic/jetpack/pull/45411).